angular-workspace/projects/ni/nimble-angular/pipes/duration.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { DurationFormatter } from '@ni/nimble-components/dist/esm/table-column/duration-text/models/duration-formatter';
+
+/**
+ * A pipe that transforms a number of milliseconds (string/number) into a string formatted like "1 day, 2 hr, 3 min, 4.567 sec"
+ */
+@Pipe({
+    name: 'duration'
+})
+export class DurationPipe implements PipeTransform {
```

Comment:
It would be good to mention the pipe in our docs somewhere. The simplest option would be to add an Angular section to the duration column storybook (then we'd do the same for the numeric column unit pipes later). I guess another option would be a dedicated story for Angular pipes. I'd be fine with that if someone strongly prefers it but I'd vote for adding it to the table column for now.

Reply:
I'd vote for adding it to the duration column storybook as usage docs. Looks like an mdx page would be needed.

Reply:
Added mdx page for the duration column with Usage Docs/Angular Usage/Duration Pipe section.